### ISSUE TITLE
feature request: flip order of labels and data in polygon popup

### ISSUE THREAD

**Issue by JackDougherty** [JackDougherty/leaflet-maps-with-google-sheets-dev#14]
@ilyankou - currently, the order of items in polygon pop-ups is hard to parse: label, data

![community_solutions_neighorhood_scorecard](https://cloud.githubusercontent.com/assets/649719/24417115/247b5914-13b5-11e7-917f-5f40bb63585d.png)

would you consider flipping the order to data, label?

@jordanpolitz requests this for her project, so if you don't want to change the code for everyone, perhaps show us where to change it for her version

**Comment by ilyankou** [comment]
In `map.js` lines 447–456:

![screen shot 2017-03-31 at 20 54 18](https://cloud.githubusercontent.com/assets/10443203/24553385/4f2a4836-1654-11e7-967b-eb4d46ce0245.png)
 
* 447–450: skips empty property names (it splits the string by commas, and if two commas in a row are encountered, it would create an empty property)
* 451–453: checks if user-specified alias for the property exists (index 1): if so, uses it. Otherwise uses the name of the property itself (index 0). This is what comes before the colon.
* 455: adds a colon followed by the value of the property in bold (`<b>`).

What you want to do is to swap the latter two blocks (and perhaps remove the colon, bold tag, etc.)

**Comment by JackDougherty** [comment]
thanks @ilyankou
